.github/workflows/release-binaries.yml

```
@@ -120,8 +124,21 @@ jobs:
 
         # Detect necessary CMake flags
         target="${{ runner.os }}-${{ runner.arch }}"
-        echo "enable-pgo=false" >> $GITHUB_OUTPUT
-        target_cmake_flags="-DLLVM_RELEASE_ENABLE_PGO=OFF"
+
+        # The hendrikmuhs/ccache-action action does not support installing sccache
+        # on arm64 Linux.
+        if [ "$target" = "Linux-ARM64" ]; then
+          echo ccache=ccache >> $GITHUB_OUTPUT
+        else
+          echo ccache=sccache >> $GITHUB_OUTPUT
+        fi
+
+        if [ "${{ runner.os }}" = "Linux" ]; then
```

```suggestion
        if [ "$RUNNER_OS" = "Linux" ]; then
```
We don't need workflow interpolation here, and it's better to avoid it if we don't need it: https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions#using-an-intermediate-environment-variable

In general I'd prefer to remove all workflow interpolation (even in instances where it is safe) to make auditing easier, but that would require plenty of changes across all workflows.

---

Here is a patch to clean up the file: https://github.com/llvm/llvm-project/pull/120860

I'll update this patch with your suggestions too.